### PR TITLE
Add gogland-eap 1.0,163.10615.22

### PR DESCRIPTION
**Pull request by mxl**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-versions/pulls
[closed issues]: https://github.com/caskroom/homebrew-versions/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/gogland-eap.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'gogland-eap' do
+  version '1.0,163.10615.22'
+  sha256 '850722f8c217fbd39cdfca7f223f6ed27065d7c7fb0ed36ff788aba02d867e15'
+
+  url "https://download.jetbrains.com/go/gogland-#{version.after_comma}.dmg"
+  name 'Gogland EAP'
+  homepage 'https://www.jetbrains.com/go'
+
+  conflicts_with cask: 'gogland'
```
Comment (miccal) [inline Casks/gogland-eap.rb]: There is no Cask called `bogland`?

Comment (miccal) [inline Casks/gogland-eap.rb]: Ha Ha, autocorrect!

Comment (mxl) [inline Casks/gogland-eap.rb]: Not yet. I added this line for the future when `gogland` will be released.

Comment (miccal) [inline Casks/gogland-eap.rb]: Ok, thanks @mxl.

**Pull request merged**